docs/standard/microservices-architecture/implement-resilient-applications/implement-http-call-retries-exponential-backoff-polly.md

```diff
@@ -18,7 +18,7 @@ The following steps show how you can use Http retries with Polly integrated into
 
 **Reference the ASP.NET Core 2.1 packages**
 
-Your project has to be using the ASP.NET Core 2.1 packages from NuGet. You typically need the `AspNetCore` metapackage, and the extension package `Microsoft.Extensions.Http.Polly`.
+Your project has to be using the ASP.NET Core 2.1(or above) packages from NuGet. You typically need the `AspNetCore` metapackage, and the extension package `Microsoft.Extensions.Http.Polly`.
```

Comment:
If the book is now being updated to 2.2, shouldn't this (and other parts) say 2.2?

Reply:
Yes, I have updated the references in most places. However, some parts of the book refer to features introduced in .NET 2.1 and also talks about how it was implemented in earlier versions. Hence, I have retained them with text such as this one:

>`HttpClientFactory` is available since .NET Core 2.1 however we recommend you to use the latest ASP.NET Core 2.2 packages from NuGet in your project. 

I had also missed out on a few Dockerfile changes which is included in my newer commits. Let me know if it's all ok?